site/app/controllers/student/GradeInquiryController.php

```diff
@@ -183,11 +210,11 @@ public function changeGradeInquiryStatus($gradeable_id) {
             $grade_inquiry->setStatus($status);
             $this->core->getQueries()->saveRegradeRequest($grade_inquiry);
             if ($content != "") {
-                $this->core->getQueries()->insertNewRegradePost($grade_inquiry->getId(), $user->getId(), $content);
+                $regrade_post_id = $this->core->getQueries()->insertNewRegradePost($grade_inquiry->getId(), $user->getId(), $content, $gc_id);
```

Comment:
can you remove the var `regrade_post_id` if it's not used in the function?

Reply:
please refer to above discussion